packages/checkout-ui-extensions/package.json

```diff
@@ -21,6 +21,7 @@
   },
   "dependencies": {
     "@remote-ui/async-subscription": "^2.1.9",
-    "@remote-ui/core": "^2.1.9"
+    "@remote-ui/core": "^2.1.9",
+    "reselect": "^4.1.5"
```

Comment:
It looks like we are just using it for `defaultMemoize`. I know `reselect` is small, but I think it's very important to avoid dependencies with this package, since any bundle size we add here is basically unavoidable for consumers, and not something we can optimize on the host side. That particular utility is ~170 lines, assuming we use the whole thing (https://github.com/reduxjs/reselect/blob/master/src/defaultMemoize.ts); any chance we could just inline it instead?

Reply:
Yeah no problems. The PR I shared had copied it initially but we started importing it at some point and it ended up shipping.

We'll need to change that in checkout-web, so if you're ok, we'll ship it now and revise in the next version (before June Edition).

Reply:
The `reselect` library is fully tree-shakeable and our build would only include the code we pulled from that package. Just double checked that locally with both webpack and rollup. Based on that I think there'd be no difference between inlining `defaultMemoize` or keep using the one from `reselect`.